pkg/sql/table.go

```diff
@@ -650,7 +651,14 @@ func (p *planner) createSchemaChangeJob(
 	}
 	job := p.ExecCfg().JobRegistry.NewJob(jobRecord)
 	if err := job.WithTxn(p.txn).Created(ctx); err != nil {
-		return sqlbase.InvalidMutationID, nil
+		if pgErr, ok := pgerror.GetPGCause(err); ok {
+			// Ignore error if system.jobs table does not exist. This will be
```

Comment:
I'm actually not sure about this. @mjibson were you returning `nil` before specifically to avoid this issue?

Reply:
Ooh, I have a much better solution for you, @nvanbenschoten. Gimme a sec.

Reply:
I don't remember ever dealing with this. If I'm in the blame output it's probably a result of a refactoring that happened.

Reply:
It's now impossible for `system.jobs` not to exist!

Reply:
Thanks @benesch!